src/app/sessions/page.tsx

```diff
@@ -0,0 +1,52 @@
+import { Suspense } from "react";
+import { cookies } from "next/headers";
+import { redirect } from "next/navigation";
+import SessionsList from "@/src/components/SessionsList";
+
+// Server component to get user ID from cookies
+async function getUserIdFromSession(): Promise<string | null> {
+  const cookieStore = await cookies();
+  const sessionToken = cookieStore.get("stytch_session");
+  
+  if (!sessionToken?.value) {
+    return null;
+  }
+
+  try {
+    const loadBackendStytch = (await import("@/lib/loadBackendStytch")).default;
```

Comment:
Why does this need to be loaded dynamically instead of imported if we're already inside a server component?

Reply:
No real intentional reason; just working fast on a Sunday morning that thought that this would be slightly protective, but you're totally right.

I'm not entirely convinced we should ship this example into this example app, it breaks with the convention of our other apps and these are all to be rewritten shortly into a monorepo. @mbramlage-stytch would love your opinion here, should we include or remove this sessions route? I'm inclined to remove it and instead just ship the main purpose of this PR, the upgrade to 15.

https://github.com/stytchauth/stytch-nextjs-app-router-example/pull/36/commits/434e9ac3355b823a3d61586fbfdd365a01073137